crates/biome_service/src/file_handlers/html.rs

```diff
@@ -672,8 +672,8 @@ fn format_embedded(
         let mut iter = embedded_nodes.iter();
         let node = iter.find(|node| node.range == range)?;
 
-        let wrap_document = |document: Document| {
-            if indent_script_and_style {
+        let wrap_document = |document: Document, is_astro_frontmatter: bool| {
```

Comment:
Let's not specialise the name of the variable. Since the second argument does influence the indentation, we should call it `should_indent` (and probably invert the Boolean condition)